src/routes/popup/send/confirm.tsx

```diff
@@ -334,6 +335,11 @@ export default function Confirm({ tokenID, qty }: Props) {
             content: browser.i18n.getMessage("sent_tx"),
             duration: 2000
           });
+          trackEvent(EventType.TX_SENT, {
+            contact: contact ? true : false,
+            amount: transactionAmount,
+            fee: networkFee
+          });
```

Comment:
we'll need to trackEvent on fallback case as well down on line ~400.
transaction amount from what i remember could be AR or other token types, If we're only monitoring AR transfers, we could have a check for `token === "AR" ? transactionAmount : 0`